lib/matplotlib/axes/_axes.py

```diff
@@ -4914,6 +4922,13 @@ def fill_betweenx(self, y, x1, x2=0, where=None,
                 for filling between two sets of y-values
 
         """
+        if not rcParams['_internal.classic_mode']:
+            color_aliases = mcoll._color_aliases
+            kwargs = cbook.normalize_kwargs(kwargs, color_aliases)
+
+            if not any(c in kwargs for c in ('color',) + tuple(color_aliases)):
+                fc = self._get_patches_for_fill.get_next_color()
+                kwargs['facecolors'] = fc
```

Comment:
Is it worth factoring out this repeated block?  Maybe that's a different project--fill_between and fill_betweenx are almost identical twins.  It seems like it should be possible to factor out almost all of their functionality, although maybe that would reduce readability.